tailwind.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  theme: {
+    fontFamily: {
+      body: ['Inter', 'sans-serif'],
+    }
+  },
+  variants: {},
+  plugins: [],
```

Comment:
This is our config for Natives in Tech:
```
module.exports = {
  purge: ["./components/**/*.tsx", "./pages/**/*.tsx"],
  theme: {
    extend: {},
  },
  variants: {},
  plugins: [require("@tailwindcss/typography")],
};
```
I think `purge` is probably the most important but if you want nice typography then that plugin is nice too.

Reply:
Good call, I think I'll add typography at least!